packages/assets-controllers/src/NftController.ts

```diff
@@ -617,9 +617,12 @@ export class NftController extends BaseControllerV1<NftConfig, NftState> {
     });
 
     const [blockchainContractData, openSeaContractData]: [
-      Partial<ApiNftContract> &
-        Pick<ApiNftContract, 'address'> &
-        Pick<ApiNftContract, 'collection'>,
+      (
```

Comment:
Is there any reason to assign a type here? It seems like TypeScript could infer this.

Reply:
Good point.  Yes, it can be inferred.